cmd/otelopscol/main.go

```diff
@@ -57,3 +69,55 @@ func runInteractive(params service.Parameters) error {
 
 	return nil
 }
+
+func setConfigEnvVars() error {
```

Comment:
The whole method is about setting one environment variable at the moment, but the error above will be just "failed to build environment variables for config".

Might be worth extracting the logic below into `setUserAgentEnvVar()` function and wrap errors returned by it?